include/swift/Parse/Lexer.h

```diff
@@ -374,13 +374,17 @@ class Lexer {
     enum : char { Literal, Expr } Kind;
     // Loc+Length for the segment inside the string literal, without quotes.
     SourceLoc Loc;
-    unsigned Length;
-    
-    static StringSegment getLiteral(SourceLoc Loc, unsigned Length) {
+    unsigned Length, Modifiers, IndentToStrip;
+
```

Comment:
nitpick: what is Modifiers for? If this is an option set of the enum below, please make this an OptionSet<> (or a type alias thereof). Similarly, plumbed throughout.